migrations/Version202112231326462260_taoQtiTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoQtiTest\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\oatbox\event\EventManager;
+use oat\oatbox\reporting\Report;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoQtiTest\models\classes\event\ResultTestVariablesTransmissionEvent;
+use oat\taoQtiTest\models\classes\evnetHandler\ResultTransmissionEventHandler\Api\ResultTransmissionEventHandlerInterface;
+use oat\taoQtiTest\models\event\ResultItemVariablesTransmissionEvent;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202112231326462260_taoQtiTest extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $eventManager = $this->getEventManager();
+        $this->attachResultItemVariablesTransmissionEvent($eventManager);
+        $this->attachResultTestVariablesTransmissionEvent($eventManager);
+        $this->getServiceManager()->register(EventManager::SERVICE_ID, $eventManager);
+    }
+
+    public function down(Schema $schema): void
+    {
+        $eventManager = $this->getEventManager();
+        $this->detachResultItemVariablesTransmissionEvent($eventManager);
+        $this->detachResultTestVariablesTransmissionEvent($eventManager);
+        $this->getServiceManager()->register(EventManager::SERVICE_ID, $eventManager);
+    }
+
+    private function getEventManager(): EventManager
+    {
+        return $this->getServiceLocator()->get(EventManager::class);
+    }
+
+    private function attachResultItemVariablesTransmissionEvent(EventManager $eventManager): void
+    {
+        $eventManager->attach(
+            ResultItemVariablesTransmissionEvent::class,
+            [ResultTransmissionEventHandlerInterface::SERVICE_ID, 'transmitResultItemVariable']
+        );
+
+        $this->addReport(
+            Report::createSuccess(
+                sprintf(
+                    'Unsubscribed %s::%s to %s event',
```

Comment:
Shouldn't be like that?
```suggestion
                    'Subscribed %s::%s to %s event',
```